sdk/search/azure-search-documents/src/test/java/com/azure/search/documents/models/IndexBatchExceptionTests.java

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents.models;
+
+import com.azure.search.documents.SearchDocument;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.mockito.Mockito.mock;
+
+public class IndexBatchExceptionTests {
+    private static final String KEY_FIELD_NAME = "key";
+    private IndexDocumentsResult result;
+
+    @BeforeEach
+    public void setup() {
+        result = mock(IndexDocumentsResult.class);
```

Comment:
You need to initialize mocking.

`MockitoAnnotations.initMocks(this);`

Reply:
Is this needed? As I do not initialize the mock through annotation.

```
Initializes objects annotated with Mockito annotations for given testClass: @Mock, @Spy, @Captor, @InjectMocks
```